File: Color_map.py
class Color_map:
    def __init__(self,data_matrix):

        self.__color_data_matrix = data_matrix

    
    def get_color_data_matrix(self):
        return self.__color_data_matrix
    
    def set_color(self, y_ind, x_ind, new_color):
        self.__color_data_matrix[y_ind, x_ind] = new_color

File: color_map.m
classdef color_map
    %COLOR_MAP Summary of this class goes here
    %   Detailed explanation goes here
    
    properties
        color_data_matrix
    end
    
    methods
        function obj = color_map(data_matrix)
            %COLOR_MAP Construct an instance of this class
            %   Detailed explanation goes here
            if nargin > 0
            obj.color_data_matrix = data_matrix;
            end
        end
    end
end

